docs/DOC_WRITING_GUIDELINES.md

```diff
@@ -8,6 +8,7 @@
 + Don't abuse `code` format when writing in plain English.
 + Follow Google developer documentation [style guide](https://developers.google.com/style).
 + Check the meaning of words in Microsoft's [A-Z word list and term collections](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms) (use the search input!).
++ Use RFC keywords: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL. They are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
```

Comment:
I totally agree that we should use these definitions for ADRs, but less sure about docs in general. It might read too much as a formal spec, whereas docs should keep a pleasant side.

My proposal is to remove these definitions for docs, only keep for ADRs.

Reply:
```suggestion
+ It is recommeneded to use RFC keywords: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL. They are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
```

maybe just a recommendation?

Reply:
I rephrased it. Could you re-check?